test/integration/targets/docker_network/tasks/tests/options.yml

```diff
@@ -92,3 +92,93 @@
     - driver_options_3 is not changed
     - driver_options_4 is changed
     - driver_options_5 is not changed
+
+####################################################################
+## scope ###########################################################
+####################################################################
+
+- block:
+  - name: scope
+    docker_network:
+      name: "{{ nname_1 }}"
+      driver: bridge
+      scope: local
+    register: scope_1
+
+  - name: scope (idempotency)
+    docker_network:
+      name: "{{ nname_1 }}"
+      driver: bridge
+      scope: local
+    register: scope_2
+
+  - name: swarm
+    docker_swarm:
+      state: present
+      advertise_addr: "{{ansible_default_ipv4.address}}"
+
+  - name: scope (change)
+    docker_network:
+      name: "{{ nname_1 }}"
+      driver: overlay
+      scope: swarm
+    register: scope_3
+
+  - name: cleanup network
+    docker_network:
+      name: "{{ nname_1 }}"
+      state: absent
+      force: yes
+
+  - assert:
+      that:
+      - scope_1 is changed
+      - scope_2 is not changed
+      - scope_3 is changed
+
+  always:
+  - name: cleanup swarm
+    docker_swarm:
+      state: absent
+      force: yes
+
+  # Requirements for docker_swarm
+  when: docker_py_version is version('2.6.0', '>=') and docker_api_version is version('1.35', '>=')
+
+####################################################################
+## attachable ######################################################
+####################################################################
+
+- block:
+  - name: attachable
+    docker_network:
+      name: "{{ nname_1 }}"
+      attachable: true
+    register: attachable_1
+
+  - name: attachable (idempotency)
+    docker_network:
+      name: "{{ nname_1 }}"
+      attachable: true
+    register: attachable_2
+
+  - name: attachable (change)
+    docker_network:
+      name: "{{ nname_1 }}"
+      attachable: false
+    register: attachable_3
+
+  - name: cleanup
+    docker_network:
+      name: "{{ nname_1 }}"
+      state: absent
+      force: yes
+
+  - assert:
+      that:
+      - attachable_1 is changed
+      - attachable_2 is not changed
+      - attachable_3 is changed
+
+  # Requirements for attachable parameter
+  when: docker_py_version is version('2.0.0', '>=') and docker_api_version is version('1.25', '>=')
```

Comment:
How about making the test also work for earlier docker-py versions (similar to the `docker_container` tests), i.e. that the minimum version check is also tested?